backend/compact-connect/lambdas/python/purchases/handlers/privileges.py

```diff
@@ -87,6 +87,8 @@ def get_purchase_privilege_options(event: dict, context: LambdaContext):  # noqa
         if item['type'] == JURISDICTION_TYPE:
             serlialized_options.append(JurisdictionOptionsResponseSchema().load(item))
         elif item['type'] == COMPACT_TYPE:
+            # we determine at run-time if the payment processor is running in sandbox mode
+            item['isSandbox'] = True if config.environment_name != 'prod' else False
```

Comment:
nit: small preference for this.

```suggestion
            item['isSandbox'] = config.environment_name != 'prod'
```